checker/tests/nullness/java8inference/Issue980.java

```diff
@@ -24,6 +24,8 @@ void m(List<String> strings) {
         // This assignment issues a warning due to incompatible types:
```

Comment:
It appears that the warning went away when the conservative type argument option was added in #1065.  (This code should not issue a warning, I think that's what this issue was about originally.)  I'll correct the comment.